doc/rados/operations/operating.rst

```diff
@@ -352,7 +352,26 @@ To stop a Ceph daemon on another node, use the following syntax::
 	sudo service ceph -a stop osd.0
 
 
+Stopping an Entire Cluster
+==========================
+
+To stop an entire ceph cluster (for example, if you must physically move
+hosts in a data center) first set the noout flag to prevent OSDs from being
+removed::
+	
+	ceph osd set noout
+
+Then shutdown all the services in your cluster in the manner appropriate for your platform.
 
+.. note:: Ceph is not really intended to be entirely shutdown, and it's behaviors in this
```

Comment:
>Ceph is not really intended to be entirely shutdown

I would tone town this statement a little bit -- Ceph *is* intended to function properly when entirely shut down, it just isn't quite as friendly as it could be.